docs/constants/navigation.js

```diff
@@ -429,7 +429,13 @@ const sections = [
   },
   {
     name: 'Expo Modules',
-    reference: ['Overview', 'Module API', 'AppDelegate Subscribers', 'Module Config'],
+    reference: [
+      'Overview',
+      'Module API',
+      'AppDelegate Subscribers',
```

Comment:
maybe we should call this `iOS AppDelegate Subscribers` given that we include `Android` in the `Lifecycle Listeners` title